libsolidity/interface/OptimiserSettings.h

```diff
@@ -43,7 +43,7 @@ struct OptimiserSettings
 			"eul"                      // Run functional expression inliner
 			"xarulrul"                 // Prune a bit more in SSA
 			"xarrcL"                   // Turn into SSA again and simplify
-			"gvif"                     // Run full inliner
+			"gvFif"                    // Run full inliner
```

Comment:
Not sure about where to place `F`. Also, `F` will cause duplication in names (arguments and return parameters)

TODO: need better tests for full suite, especially recursive functions.